### PR TITLE
Updated colors for table in dark mode

### PR DESCRIPTION
**Pull request by cheepo2109**
## Hey, I just made a Pull Request for issue #742

I've updated colors in dark mode to match requirements in the above mentioned issue

CleanShot 2020-05-10 at 01 32 53@2x

CleanShot 2020-05-10 at 01 33 24@2x

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/core/src/components/Table/Table.tsx [hunk 1]
```diff
@@ -130,13 +130,14 @@ const useToolbarStyles = makeStyles<BackstageTheme>(theme => ({
 }));
 
 const convertColumns = (columns: TableColumn[]): TableColumn[] => {
-  return columns.map(column => {
+  return columns.map((column) => {
+    const theme = useTheme<BackstageTheme>();
```
Comment (Rugvip) [inline packages/core/src/components/Table/Table.tsx:134]: Hooks should always be called unconditionally and directly within the scope of a component function or another custom  hook. You want to call this hook in the component and then pass it in to this function instead.

File: packages/core/src/components/Table/Table.tsx [hunk 2]
```diff
@@ -99,26 +99,26 @@ const tableIcons = {
...
 
-const useHeaderStyles = makeStyles<BackstageTheme>(theme => ({
+const useHeaderStyles = makeStyles<BackstageTheme>((theme) => ({
   header: {
     padding: theme.spacing(1, 2, 1, 2.5),
     borderTop: `1px solid ${theme.palette.grey.A100}`,
     borderBottom: `1px solid ${theme.palette.grey.A100}`,
     color: theme.palette.textSubtle,
-    fontWeight: 'bold',
+    fontWeight: theme.typography.fontWeightBold,
```
Comment (Rugvip) [inline packages/core/src/components/Table/Table.tsx:116]: 👌

**Review by Rugvip — COMMENTED** [review]
Thanks!

File: packages/core/src/components/Table/Table.tsx (hunk 1 above)
Comment (cheepo2109) [reply packages/core/src/components/Table/Table.tsx:134]: Nice catch! Gonna update it in a bit :)

File: packages/core/src/components/Table/Table.tsx [hunk 3]
```diff
@@ -160,8 +162,9 @@ const Table: FC<TableProps> = ({ columns, options, ...props }) => {
   const cellClasses = useCellStyles();
   const headerClasses = useHeaderStyles();
   const toolbarClasses = useToolbarStyles();
+  const theme = useTheme<BackstageTheme>();
```
Comment (cheepo2109) [inline packages/core/src/components/Table/Table.tsx:165]: @Rugvip updated :)

File: packages/theme/src/themes.ts [hunk 4]
```diff
@@ -60,7 +60,7 @@ export const lightTheme = createTheme({
 export const darkTheme = createTheme({
   type: 'dark',
   background: {
-    default: '#282828',
+    default: '#333333',
```
Comment (Rugvip) [inline packages/theme/src/themes.ts:63]: Noting that this is a change to the theme, but I think it's what we want

**Review by Rugvip — APPROVED** [review]

**Review by stefanalund — APPROVED** [review]

**Comment by stefanalund** [comment]
Great job, thanks a lot!

**Pull request merged**